13_sandboxes/test_case_generator.py

```diff
@@ -0,0 +1,274 @@
+# ---
+# cmd: ["modal", "run", "-m", "13_sandboxes.test_case_generator::main"]
+# ---
+import modal
+
+app = modal.App(name="sandbox-test-case-generator")
+model_volume = modal.Volume.from_name("deepseek-model-volume", create_if_missing=True)
+files_volume = modal.Volume.from_name("files-volume", create_if_missing=True)
+
+
+model_image = (
+    modal.Image.from_registry("nvidia/cuda:12.4.0-devel-ubuntu22.04", add_python="3.12")
+    .pip_install(
+        "transformers==4.53.2",
+        "torch==2.7.1",
+        "accelerate==1.8.1",
+        "hf_transfer==0.1.9",
+    )
+    .env(
+        {
+            "HF_HUB_ENABLE_HF_TRANSFER": "1",
+            "HF_HOME": "/cache",
+        }
+    )
+    .entrypoint([])  # silence noisy logs
+)
+
+with model_image.imports() as imports:
+    from transformers import AutoModelForCausalLM, AutoTokenizer
+
+
+# GH Repo Configs
```

Comment:
Can this config be passed from a command-line interface?

Otherwise this example requires code changes to change the target repo + branch.